bookman

```diff
@@ -89,16 +89,16 @@ if [ -n "$SOURCE_DATE_EPOCH" ]; then
 fi
 date=${date:-$(date +'%d %B %Y')}
 
-[[ $1 ]] || set -- $(while read; do echo $REPLY; done)
```

Comment:
agree to replace `[[`by `[` and be POSIX compatible. While there, could you enclose variables in double quotes to tolerate spaces or hyphens etc. i.e. `[ "$1" ] || set -- $(while read REPLY; do echo "$REPLY"; done)``?
Thanks